presto-druid/src/test/java/io/prestosql/plugin/druid/TestDruidSimpleQueries.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.druid;
+
+import io.prestosql.tests.AbstractTestQueryFramework;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.Test;
+
+@Test
+public class TestDruidSimpleQueries
+        extends AbstractTestQueryFramework
+{
+    private final TestingDruidServer druidServer;
+
+    public TestDruidSimpleQueries()
+    {
+        this(new TestingDruidServer());
+    }
+
+    protected TestDruidSimpleQueries(TestingDruidServer testingDruidServer)
+    {
+        super(() -> DruidQueryRunner.createDruidQueryRunner(testingDruidServer));
+        this.druidServer = testingDruidServer;
+    }
+
+    @Test
+    public void testOrderByWithSimilarExpressions()
+    {
+        assertQuery(
+                "WITH t AS (SELECT 1 x, 2 y) SELECT x, y FROM t ORDER BY x, y",
+                "SELECT 1, 2");
+        assertQuery(
+                "WITH t AS (SELECT 1 x, 2 y) SELECT x, y FROM t ORDER BY x, y LIMIT 1",
+                "SELECT 1, 2");
+        assertQuery(
+                "WITH t AS (SELECT 1 x, 1 y) SELECT x, y FROM t ORDER BY x, y LIMIT 1",
+                "SELECT 1, 1");
```

Comment:
these queries do not touch connector code. Please remove them from here and below.